src/mapping/sanity/join-us.tsx

```diff
@@ -1,4 +1,60 @@
-// fetch data with graphql query from sanity
-// fetch component and pass data to the component
-// remove data source from component
-export {}
+import { JoinUs } from '@src/components/features/decoupled-components/join-us/join-us';
+import {
+  JoinUsClient,
+  JoinUsItem,
+  JoinUsProps,
+} from '@src/components/features/decoupled-components/join-us/join-us.typings';
+
+interface JoinUsClientItemsProps {
```

Comment:
Can all typing related data be put into a specific file? Mapping should use the mapping that is attached to the component or the data. It's not the proper place to define structure, mappings should link existing structures.